renovate.json

```diff
@@ -1,6 +1,31 @@
 {
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "local>kaliber5/renovate-config:ember-addon"
```

Comment:
kaliber5/renovate-config is not well maintained anymore. And it is configured for yarn. I have inlined the configuration and changed the `postUpdateOptions` to support pnpm.